src/Opaleye/Internal/Print.hs

```diff
@@ -73,6 +75,14 @@ ppSelectLabel l = text "/*" <+> text (defuseComments (Sql.lLabel l)) <+> text "*
                    . ST.replace (ST.pack "*/") (ST.pack " * / ")
                    . ST.pack
 
+ppSelectAntijoin :: Antijoin -> Doc
+ppSelectAntijoin v =
+  text "SELECT *"
+  $$ text "FROM"
+  $$ ppTable (tableAlias 1 (Sql.antijoinTable v))
```

Comment:
Is it really the case that the first clause requires a table alias and the second doesn't?  If so that's very strange on the part of PostgreSQL!